psamm/datasource/sbml.py

```diff
@@ -713,20 +732,40 @@ def write_model(self, file, model, reactions, compounds):
 
         # Create list of reactions
         reactions = ET.SubElement(model_tag, self._sbml_tag('listOfReactions'))
-        for reaction in model.reactions:
+        # Create mapping for reactions containing flux limit definitions
+        for flux_limit in model.parse_limits():
```

Comment:
You can rewrite this loop to `for reaction_id, lower, upper in model.parse_limits():...` to give the more clear variable names instead of `flux_limits[0]`, `flux_limits[1]`, etc.